### ISSUE TITLE
requests.exceptions.HTTPError: 415 Client Error: Unsupported Media Type for url

### ISSUE THREAD

**Issue by benwoodward** [bwilczynski/sonos-cli#9]
```
❯ sonos pause
Traceback (most recent call last):
  File "/usr/local/bin/sonos", line 8, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/sonos/decorators.py", line 58, in wrapper_login_required
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/sonos/commands/pause.py", line 12, in pause
    control.pause(group_id)
  File "/usr/local/lib/python3.7/site-packages/sonos/decorators.py", line 80, in wrapper_auto_refresh_token
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/sonos/api/control.py", line 51, in pause
    return _json(response)
  File "/usr/local/lib/python3.7/site-packages/sonos/api/control.py", line 20, in _json
    response.raise_for_status()
  File "/usr/local/lib/python3.7/site-packages/requests/models.py", line 941, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 415 Client Error: Unsupported Media Type for url: https://api.ws.sonos.com/control/api/v1/groups/RINCON_omitted:omitted/playback/pause
```

Currently streaming Spotify to Sonos One

**Comment by bwilczynski** [comment]
Fixed in sonos-cli 0.3.5

**Comment by benwoodward** [comment]
Fantastic, it works! Great work, thank you @bwilczynski !
